src/Components/NavBar/NavBar.tsx

```diff
@@ -422,15 +422,6 @@ export const NavBar: React.FC = track(
                 >
                   {t`navbar.museums`}
                 </NavBarItemLink>
-
-                <NavBarItemLink
-                  href="/collect?additional_gene_ids[0]=nft"
-                  onClick={handleClick}
-                  data-label="NFTs"
-                  display={["none", "none", "flex"]}
-                >
-                  {t`navbar.nfts`}
-                </NavBarItemLink>
```

Comment:
👋

Reply:
that was removed, was it in the wrong place??

Reply:
I think he was waving goodbye :p

Reply:
oh haha yea adios, NFTs!